test/framework/module_generator.py

```diff
@@ -139,12 +143,15 @@ def test_load(self):
             ])
             self.assertEqual(expected,self.modgen.load_module("mod_name"))
 
-            init_config(build_options={'recursive_mod_unload': True})
+            # with recursive unloading: no if isloaded guard
             expected = '\n'.join([
                 '',
                 'load("mod_name")',
                 '',
             ])
+            self.assertEqual(expected, self.modgen.load_module("mod_name", recursive_unload=True))
+
+            init_config(build_options={'recursive_mod_unload': True})
```

Comment:
This shouldn't be around any more should it, it's triggering the same thing twice?